main.py

```diff
@@ -73,17 +74,55 @@ def move_player(player_pos, keys, paths, speed):
             current_orientation = 'vertical'
     return current_orientation
 
-def ask_question_node():
+# Custom dialog box for asking questions
+class QuestionDialog(simpledialog.Dialog):
```

Comment:
create a new folder for this class and import it into this main file, this helps to keep the main.py file clean and loosely coupled with anyones code. If anything fails then we would know that the issue is with the imported module and not the main file. We want to keep the main file simple and clean because it is the heart of this project

Reply:
Im not sure how to do that

Reply:
@kanan42  You just need to write your class in a new Python script and save it. After that, you can just import your class in any script you need. Peter made a class as an example it's in the utils folder.